js/main.js

```diff
@@ -125,11 +135,35 @@
     addressField.value = getCurrentMainPinPosition();
   }
 
-  disableForms(); // Пока вхолостую, чтобы линтер не ругался
+  function onTypeChange(evt) {
+    evt.preventDefault();
+    setMinPriceByType();
+  }
+
+  function onTimeinTimeoutChange(evt) {
+    evt.preventDefault();
+    timeinField.value = evt.target.value;
+    timeoutField.value = evt.target.value;
+  }
+
+  // Устанавливает минимальную стоимость жилья
+  function setMinPriceByType() {
+    var type = typeField.value;
+    priceField.setAttribute('min', MIN_PRICE[type]);
```

Comment:
```suggestion
    priceField.setAttribute('min', MIN_PRICE[typeField.value]);
```
простенько и со вкусом:)

Reply:
вот смотрю и чувствую себя глупо)